### DISCUSSION TITLE
python3.9 compile

### DISCUSSION THREAD

**Discussion by derrij** [Entware/Entware#811] [Q&A] [answered]
Hi. How can I compile entware with python3.9? How can i get back to 2021/08/28 version?
https://entware.net/2021/08/28/Changelog.html
Thank you.

**Comment by derrij** [comment, marked as answer]
after 
```
cd feeds/packages
git fetch --depth 1 origin 79a2026f36b36651ecbbb02869a5e145c530adca
git checkout FETCH_HEAD
```
i can't compile all boost modules.

Or maybe i can get somewhere  Aug 28, 2021 bin repo?

**Reply by ryzhovau** [reply]
We are on modified OpenWrt Buildroot, you may follow to [OpenWrt docs](https://openwrt.org/docs/guide-developer/toolchain/use-buildsystem?s[]=buildroot).

You can always find any previous version of every package at `http://bin.entware.net/<target>/archive/`

**Comment by ryzhovau** [comment]
Different packages are dependent from each other. So you have to set the same date as for buildroot, as for any other feeds.
I think it's the only way to get repo as it was in Aug 28, 2001.
